docs/scriptappy.json

```diff
@@ -4762,6 +4762,37 @@
           "description": "Bounding rectangle of the node withing it's local coordinate system.\nOrigin is in the top-left corner of the scene element.",
           "type": "#/definitions/rect"
         },
+        "boundsRelativeTo": {
+          "description": "Bounding rectangle of the node, relative a target.\n\nIf target is an HTMLElement, the bounds is relative to the HTMLElement.\nAny other target type will return the bounds reltive to the viewport of the browser.",
+          "kind": "function",
+          "params": [
+            {
+              "name": "target",
+              "kind": "union",
+              "items": [
+                {
+                  "type": "HTMLElement"
+                },
+                {
+                  "type": "any"
+                }
+              ],
+              "type": "any"
+            },
+            {
+              "name": "includeTransform",
+              "description": "true will include any transform attribute on the node, false will not",
```

Comment:
Can we have a description that is more in line with the convention of describing booleans,like: _Whether to include any applied transforms on the node_.